src/toil/wdl/wdltoil.py

```diff
@@ -767,13 +852,13 @@ def __init__(self, file_store: AbstractFileStore, execution_dir: Optional[str] =
         # paths, to save re-uploads.
         self._devirtualized_to_virtualized: Dict[str, str] = {}
 
-        self._execution_dir = execution_dir
-
-        self._enforce_existence = enforce_existence
+        self._wdl_options = wdl_options
 
     @property
     def execution_dir(self) -> Optional[str]:
-        return self._execution_dir
+        wdl_options = self._wdl_options or {}
+        execution_dir: Optional[str] = wdl_options.get("execution_dir")
```

Comment:
Maybe instead of just a dict and a convention we want this to be a TypedDict or even a class? Then MyPy will catch if we e.g. misspell `execution_dir` somewhere, and we can go to they type definition to see all the Toil WDL interpreter options we support and what they mean.